github.py

```diff
@@ -99,14 +99,15 @@ def get(self, url, headers=None):
         resp.raise_for_status()
         return resp
 
-    def post_comment(self, issue_number, body, title):
+    def post_comment(self, issue_number, body, title, full_log_url):
         """Create or update comment in pull request comment section."""
         user = self.get(urljoin(self.base_url, "/user")).json()
         issue_comments_url = urljoin(self.base_url,
                                      "issues/%s/comments" % issue_number)
         comments = self.get(issue_comments_url).json()
         title_line = format_comment_title(title)
-        data = {"body": body}
+        link = ' [View complete log on Travis CI.](%s)' % full_log_url
```

Comment:
It's a slight violation of our separation of concerns for the "github" module to reference "Travis CI" specifically. In the original patch, I just referred to "the complete log" generically.